package.json

```diff
@@ -53,7 +53,8 @@
     "lodash": "^4.17.5",
     "lodash-es": "^4.17.5",
     "loose-envify": "^1.1.0",
-    "prop-types": "^15.6.1"
+    "prop-types": "^15.6.1",
+    "react-lifecycles-compat": "^1.1.4"
```

Comment:
Why do you need the polyfill when you changed all the deprecated functions?

Reply:
He's using `getDerivedStateFromProps`.

Reply:
I am using that too without a polyfill. No more warnings (in my own code).

Reply:
It's for use with older versions of React. It's not to deal with deprecation warnings, it's to polyfill those lifecycle methods for React <= 16.2.

Reply:
Ah right, didn't think it that way around.
That brings me to a question for my own lib. What if I want to support the new and old Context API I guess I could just check if `React.createContext` is a function and then use it if avail?

Reply:
@pke : there's a distinct difference in how you declare and use the old and new context APIs.  See my PR in #898 for an example.

Reply:
New context API is probably going to require breaking changes to React Redux's public API for supporting multiple stores, making it a major version bump.